applications/GeoMechanicsApplication/custom_elements/updated_lagrangian_U_Pw_axisymmetric_FIC_element.hpp

```diff
@@ -42,11 +42,12 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwUpdatedLagrangianAxisymmetricFICE
     /// The definition of the sizetype
     using SizeType = std::size_t;
     using UPwBaseElement<TDim, TNumNodes>::mConstitutiveLawVector;
+    using typename UPwSmallStrainElement<TDim, TNumNodes>::ElementVariables;
```

Comment:
This line would not be needed if we would pass `const Matrix&` (representing the deformation gradient) to `CalculateGreenLagrangeStrain`.